src/main/java/uk/gov/hmcts/ccd/data/casedetails/CaseAuditEventRepository.java

```diff
@@ -41,10 +42,13 @@ public AuditEvent set(final AuditEvent auditEvent) {
     }
 
     public List<AuditEvent> findByCase(final CaseDetails caseDetails) {
-        final Query query = em.createNamedQuery(CaseAuditEventEntity.FIND_BY_CASE);
-        query.setParameter(CaseAuditEventEntity.CASE_DATA_ID, Long.valueOf(caseDetails.getId()));
+        final List<CaseAuditEventEntity> resultList = em.createNamedQuery(CaseAuditEventEntity.FIND_BY_CASE)
+            .setParameter(CaseAuditEventEntity.CASE_DATA_ID, Long.valueOf(caseDetails.getId()))
+            .unwrap(org.hibernate.query.Query.class)
+            .setResultTransformer(Transformers.aliasToBean(CaseAuditEventEntity.class))
+            .getResultList();
```

Comment:
I think Query. setResultTransformer(..) is deprecated. Check if any alternatives provided in the latest version.

Reply:
As per this link (https://discourse.hibernate.org/t/hibernate-resulttransformer-is-deprecated-what-to-use-instead/232) this will not be replaced with a new method until version 6 of Hibernate.